modules/features/falcon_feature_user/config/install/user.role.authenticated.yml

```diff
@@ -0,0 +1,14 @@
+langcode: en
+status: true
+dependencies: {  }
+id: authenticated
+label: 'Authenticated user'
+weight: 1
+is_admin: false
+permissions:
```

Comment:
Neither of given commerce permissions seem to be required

Reply:
Removed